tools/helper/create_temp_account.hpp

```diff
@@ -0,0 +1 @@
+//TODO                                txbuilder::createAccount( argv[0], argv[1], txbuilder::createTrust( stod(argv[2]), argv[3]=="true" ) )
```

Comment:
Aer there this files other directory?
for example)
`tools/helper/createObject/*`